src/header/header.scss

```diff
@@ -290,7 +302,7 @@
             padding: $iui-xxs;
           }
 
-          .iui-label {
+          .iui-button-label {
```

Comment:
This doesn't need to be nested inside .iui-button anymore.

Reply:
removed nesting